x-pack/platform/plugins/shared/streams/server/routes/streams/enablement/route.ts

```diff
@@ -60,9 +59,7 @@ export const disableStreamsRoute = createServerRoute({
   },
   security: {
     authz: {
-      enabled: false,
-      reason:
-        'This API delegates security to the currently logged in user and their Elasticsearch permissions.',
+      requiredPrivileges: [STREAMS_FEATURE_PRIVILEGES.assets, STREAMS_FEATURE_PRIVILEGES.write],
```

Comment:
We can use one or the other, since they are both only available to the "all" privilege access

Reply:
Pattern for this would be: `requiredPrivileges: [{ anyRequired: ['<privilege_1>', '<privilege_2>'] }],`

Reply:
I agree, but I wonder if we need to have these two privileges since they are only given when the feature access control is set to "All". The user either has these two privileges, or none. But this also goes back to my other thought: do we need both assets and write privileges or just a "manage_stream" privilege is enough?

Reply:
I left it in because I thought it's better to be explicit so we can give more fine-grained permissions later on, but that's actually not a good reason to do things. I will keep it simple and just go for read and manage

Reply:
Removed `assets`